website_blog_rss/views/website_templates.xml

```diff
@@ -3,21 +3,26 @@
     <data>
     <template id="blog_rss_locs">
         <channel>
-        <item t-foreach="posts" t-as="post">
-            <t t-set="url_x" t-value="'{0}blog/{1}/post/{2}'.format(url_root, slug(post.blog_id), slug(post))"/>
-            <title><t t-esc="post.name"/></title>
-            <link><t t-raw="url_x"/></link>
-            <lastBuildDate><t t-esc="post.write_date"/></lastBuildDate>
-            <description><t t-esc="post.content"/></description>
-            <category><t t-esc="post.blog_id.name"/></category>
-        </item>
+        <title><t t-esc="company.name"/></title>
+        <link><t t-esc="company.website"/></link>
+        <description><t t-esc="company.rml_header1"/></description>
+        <t t-foreach="posts" t-as="post">
+                <t t-set="url_x" t-value="'{0}blog/{1}/post/{2}'.format(url_root, slug(post.blog_id), slug(post))"/>
+            <item>
+                <title><t t-esc="post.name"/></title>
+                <link><t t-raw="url_x"/></link>
```

Comment:
It would be nice to have a TTL too (probably a new field to add on blog.blog) -> this optimizes aggregators and helps reduce the load on the server.